pkg/app/app.go

```diff
@@ -65,41 +65,40 @@ func ValidateFlags(bundle string, args []string, cmd *cobra.Command, opt *kobjec
 	logrus.Debug("Checking validation of provider %s", provider)
 
 	// OpenShift specific flags
-	deploymentConfig := cmd.Flags().Lookup("deployment-config").Changed
```

Comment:
this is not required, because opt is single point of truth. If a user is not providing flag then default controller is chosen, see comment in https://github.com/kubernetes-incubator/kompose/pull/366#discussion_r95106896